I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 104, in script
    with Fade()
TypeError: Fade() missing 3 required positional arguments: 'out_time', 'hold_time', and 'in_time'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 104, in script
    with Fade()
  File "C:\Users\nathan\Downloads\renpy-8.2.1-sdk\renpy\ast.py", line 1363, in execute
    renpy.exports.with_statement(trans, paired=paired)
  File "C:\Users\nathan\Downloads\renpy-8.2.1-sdk\renpy\exports.py", line 1781, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "C:\Users\nathan\Downloads\renpy-8.2.1-sdk\renpy\display\core.py", line 1548, in do_with
    return self.interact(trans_pause=True,
  File "C:\Users\nathan\Downloads\renpy-8.2.1-sdk\renpy\display\core.py", line 2165, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "C:\Users\nathan\Downloads\renpy-8.2.1-sdk\renpy\display\core.py", line 2608, in interact_core
    trans = instantiate_transition(None, old_root, layers_root)
  File "C:\Users\nathan\Downloads\renpy-8.2.1-sdk\renpy\display\core.py", line 2518, in instantiate_transition
    trans = self.ongoing_transition[layer](
TypeError: Fade() missing 3 required positional arguments: 'out_time', 'hold_time', and 'in_time'

Windows-10-10.0.19045 AMD64
Ren'Py 8.2.1.24030407
Re:Life  1.0
Sun May 26 16:41:05 2024
